packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -16,11 +16,20 @@ export interface RoleProps {
 
   /**
    * A list of ARNs for managed policies associated with this role.
-   * You can add managed policies later using `attachManagedPolicy(arn)`.
+   * You can add managed policies later using ``attachManagedPolicy(arn)``.
    * @default No managed policies.
    */
   managedPolicyArns?: string[];
 
+  /**
+   * A list of named policies to inline into this role. These policies will
+   * be created with the role, whereas those added by ``addToPolicy`` are
+   * added using a separate CloudFormation resource (allowing a way around
+   * cicrular dependencies that could otherwise be introduced).
+   * @default No inline policies.
+   */
+  policies?: { [name: string]: PolicyDocument };
```

Comment:
Maybe we should call this `inlinePolicies`

Reply:
I debated myself on this one. Changing!